NSFW and SPAM (#69)
vmstan/vmstio: docus/content/rules/spam.md

---
title: SPAM
---

# SPAM

No advertising, spam or excessive promotion.

## Advertising

> The prohibition against advertising generally applies to posts where the user is being financially compensated for their content AKA sponsored posting.

Reviewer: ```suggestion
The prohibition against advertising generally applies to posts where the user is being financially compensated for their content, also known as "sponsored posting".
```

- Crowdfunding links to content that is relevant to our users are fine.
- Additionally, users asking for donations for charities or themselves as forms of mutual-aid are not considered in violation of this rule.

vmst.io itself cannot evaluate every request for donations so we ask users use their own best judgement when evaluating whether or not to contribute to causes advocated by individual users.

## Excessive Promotion

There is no hard and fast measure for what defines excessive.

Generally the types of accounts we want on vmst.io are personal accounts where you identify as you, with your name and face. You speak in your voice, you participate in the community and the wider fediverse as you.

You can post about life, work, hobbies, ups and downs, memes, whatever you'd like (within the [rules](/rules).)

As outlined above, life and work may cross over into revenue generating activity and that's fine.
Until we reach the 23rd century Star Trek utopia where we can live our dreams without worry about how are basic needs are fulfilled, we live in societies where we have to have money to survive.

> When the bulk or your activity on the site becomes in service of that promotion, or borders on being considered a [corporate or brand account](/rules/brands), we would ask that you migrate to another instance that better fits your needs.

Reviewer: ```suggestion
When the bulk of your activity on the site becomes in service of that promotion, or borders on being considered a [corporate or brand account](/rules/brands), we would ask that you migrate to another instance that better fits your needs.
```

## Hashtags Overuse

Due to the federated nature of Mastodon, and current limitations in searching, hashtags are used to help with content delivery and discovery.
However, their overuse can be disruptive and degrade their value.

Users who misuse or overuse hashtags may be asked by instance staff to stop, or have limitations placed on their account which would prevent their non-followers from discovering their content.

> Continued misuse may result in account temporary or permanent account suspension.

Reviewer: ```suggestion
Continued misuse may result in temporary or permanent account suspension.
```
